src/components/views/right_panel/RoomHeaderButtons.js

```diff
@@ -56,8 +56,13 @@ export default class RoomHeaderButtons extends HeaderButtons {
     }
 
     _onMembersClicked() {
-        // This toggles for us, if needed
-        this.setPhase(RIGHT_PANEL_PHASES.RoomMemberList);
+        if (this.state.phase === RIGHT_PANEL_PHASES.RoomMemberInfo) {
+            // send the active phase to trigger a toggle
+            this.setPhase(RIGHT_PANEL_PHASES.RoomMemberInfo);
```

Comment:
This is asking for a bug down the line when someone tries to do something with a RoomMemberInfo phase change and wonders why there's no room member info :(

can we send the extra info for safety please?

likewise for groups.

Reply:
I'm not sure I follow what you mean by send the extra info for safety

Reply:
It should be this line:
```
this.setPhase(RIGHT_PANEL_PHASES.RoomMemberInfo, {member: payload.member});
```

except using the right `member` in the `extraInfo` (2nd arg)